### ISSUE TITLE
Bug Report: Response Status Codes as Strings Instead of Numbers

### ISSUE THREAD

**Issue by lukaskupczyk** [Carminepo2/openapi-ts-rest#89]
Thank you for this great library!

Issue: I have noticed that the response status codes are of the type string instead of number. As a result, the response body is always of the type unknown.

Reproducible Example:

OpenAPI Specification:

```yaml
openapi: "3.0.2"
info:
  title: Example


paths:
  /test:
    get:
      operationId: test
      responses:
        200:
          description: OK
          content:
            application/json:
              schema:
                type: string
```

Produces:
```typescript
import { initContract } from "@ts-rest/core";
import { z } from "zod";

const c = initContract();

export const contract = c.router({
  test: { method: "GET", path: "/test", responses: { "200": z.string() } },
});
```

Instead of:
```typescript
import { initContract } from "@ts-rest/core";
import { z } from "zod";

const c = initContract();

export const contract = c.router({
  test: { method: "GET", path: "/test", responses: { 200: z.string() } },
});
```

Note: The quotation marks around 200 should be removed.

**Comment by Carminepo2** [comment]
Hi, thank you very much for the bug report, I'm glad my library is being useful to you! :)

Gonna fix this asap

**Comment by Carminepo2** [comment]
Should be solved in version `0.1.3`. Let me know if everything is alright!

**Comment by lukaskupczyk** [comment]
Awesome. This solves the issue. Thank you very much for the swift response and fix! 🥇
